config/standalone.example.toml

```diff
@@ -457,6 +457,27 @@ mem_threshold_on_create = "64M"
 ## Deprecated, use `region_engine.mito.index.aux_path` instead.
 intermediate_path = ""
 
+## The options for full-text index in Mito engine.
+[region_engine.mito.fulltext_index]
+
+## Whether to create the index on flush.
+## - `auto`: automatically
+## - `disable`: never
+create_on_flush = "auto"
+
+## Whether to create the index on compaction.
+## - `auto`: automatically
+## - `disable`: never
+create_on_compaction = "auto"
+
+## Whether to apply the index on query
+## - `auto`: automatically
+## - `disable`: never
+apply_on_query = "auto"
+
+## Memory threshold for index creation.
```

Comment:
Is it too small? Maybe we can set the default value according to the system memory, just like 

https://github.com/GreptimeTeam/greptimedb/blob/09fff24ac4149a22f530a0e25df60c1aab0f5f73/src/mito2/src/config.rs#L145

Then the default value can be `auto`.

Reply:
Other index memory or parallelism configurations can be tweaked this way too.

Reply:
good idea!

Reply:
updated